bodhi/server/util.py

```diff
@@ -317,22 +318,53 @@ def sanity_check_repodata(myurl, repo_type):
             tests.append((['list', 'available'], 'testrepo'))
         elif repo_type == 'module':
             tests.append((['module', 'list'], '.*'))
+        else:
```

Comment:
Would we not want to run this on "yum" repos?

Reply:
Yeah, that was my bug :).